Std/Classes/LawfulMonad.lean

```diff
@@ -74,7 +74,8 @@ namespace SatisfiesM
 /-- If `p` is always true, then every `x` satisfies it. -/
 theorem of_true [Applicative m] [LawfulApplicative m] {x : m α}
     (h : ∀ a, p a) : SatisfiesM p x :=
-  ⟨(fun a => ⟨a, h a⟩) <$> x, by simp [← comp_map, Function.comp]⟩
+  ⟨(fun a => ⟨a, h a⟩) <$> x,
+    by simp (config := { unfoldPartialApp := true }) [← comp_map, Function.comp]⟩
```

Comment:
@digama0, in Mathlib we have the lemma `Function.comp_def`, and there I have used that to avoid needing `unfoldPartialApp := true` every time we want to unfold `Function.comp`.

Do you think I should move that up to Std?

Reply:
Okay, I've done this in #367 now.